src/openpersonen/api/tests/views/test_kind.py

```diff
@@ -349,6 +350,30 @@ def test_list_kind(self):
             str(self.kind.datum_ingang_onderzoek),
         )
 
+    def test_list_kind_without_birth_place(self):
+        persoon = PersoonFactory.create(burgerservicenummer_persoon=987654321)
+        kind = KindFactory(
+            persoon=persoon,
+            burgerservicenummer_kind=222222222,
+        )
+        kind.geboorteplaats_kind = ""
+        kind.geboorteland_kind = ""
+        kind.save()
+
+        response = self.client.get(
+            reverse(
+                "kinderen-list",
+                kwargs={"ingeschrevenpersonen_burgerservicenummer": 987654321},
+            ),
+            HTTP_AUTHORIZATION=f"Token {self.token.key}",
+        )
+
+        self.assertEqual(response.status_code, 200)
+
+        response_json = response.json()
+        self.assertTrue(isinstance(response_json["_embedded"]["kinderen"], list))
```

Comment:
```suggestion
        self.assertIsInstance(response_json["_embedded"]["kinderen"], list)
```

See the table below https://docs.python.org/3.8/library/unittest.html#unittest.TestCase.debug

Reply:
Aah nice!